### PR TITLE
feat/SDK-2

### PR DESCRIPTION
**Pull request by BtencateSphereon**

### CODE REVIEW

File: packages/contact-manager/src/agent/ContactManager.ts [hunk 1]
```diff
@@ -81,23 +104,34 @@ export class ContactManager implements IAgentPlugin {
...
-      uri: args.uri,
-      partyType: args.contactType,
+      uri,
+      partyType: contactType,
       contact: this.getContactInformationFrom(args),
-      identities: args.identities,
-      electronicAddresses: args.electronicAddresses,
+      identities,
+      electronicAddresses,
+      physicalAddresses, // TODO test adding this
```
Comment (nklomp) [inline packages/contact-manager/src/agent/ContactManager.ts:121]: Is the comment still applicable?

File: packages/data-store/src/entities/contact/ElectronicAddressEntity.ts [hunk 2]
```diff
@@ -32,6 +32,9 @@ export class ElectronicAddressEntity extends BaseEntity {
   })
   party!: PartyEntity
 
+  @Column({ name: 'partyId', nullable: true })
+  partyId!: string
```
Comment (nklomp) [inline packages/data-store/src/entities/contact/ElectronicAddressEntity.ts:36]: The collumn is nullable, but the partyId has an !; is that supposed to be?

File: packages/data-store/src/entities/contact/PhysicalAddressEntity.ts [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+  @IsNotEmpty({ message: 'Blank building names are not allowed' })
+  buildingName?: string
+
+  @ManyToOne(() => PartyEntity, (party: PartyEntity) => party.physicalAddresses, {
+    onDelete: 'CASCADE',
+  })
+  party!: PartyEntity
+
+  @Column({ name: 'partyId', nullable: true })
+  partyId!: string
```
Comment (nklomp) [inline packages/data-store/src/entities/contact/PhysicalAddressEntity.ts:60]: Nullable but required for TS

**Review by nklomp — CHANGES_REQUESTED** [review]
Some small remarks about potential inconsistency between the ORM layer and TS

File: packages/contact-manager/src/agent/ContactManager.ts (hunk 1 above)
Comment (BtencateSphereon) [reply packages/contact-manager/src/agent/ContactManager.ts]: removed

File: packages/data-store/src/entities/contact/ElectronicAddressEntity.ts (hunk 2 above)
Comment (BtencateSphereon) [reply packages/data-store/src/entities/contact/ElectronicAddressEntity.ts]: fixed

File: packages/data-store/src/entities/contact/PhysicalAddressEntity.ts (hunk 3 above)
Comment (BtencateSphereon) [reply packages/data-store/src/entities/contact/PhysicalAddressEntity.ts]: fixed

**Review by nklomp — APPROVED** [review]
LGTM

**Pull request merged**